tests/framework/di/InstanceTest.php

```diff
@@ -46,7 +46,27 @@ public function testEnsure()
         $this->assertTrue(Instance::ensure(['class' => 'yii\db\Connection', 'dsn' => 'test'], 'yii\db\Connection', $container) instanceof Connection);
     }
 
-    public function testEnsureWithoutType()
+    /**
+     * ensure an InvalidConfigException is thrown when a component does not exist.
```

Comment:
a readable description is much better than a method name. There is no such thing as self documenting code.